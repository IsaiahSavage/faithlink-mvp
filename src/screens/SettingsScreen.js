import React from 'react';
import { StatusBar } from 'expo-status-bar';
import { View, Text, StyleSheet, FlatList } from 'react-native';

const SettingsScreen = ({ route, navigation }) => {
  const SETTINGS = [];

  return (
    <View style={styles.contentContainer}>
      <StatusBar style="auto" />
      <Text style={styles.title}>Settings coming soon!</Text>
      {/* <FlatList
        data={settings}
        renderItem={({ item }) => <Text>{item.title}</Text>}
        keyExtractor={(item) => item.title}
        style={styles.settingsContainer}
      /> */}
    </View>
  );
};

const styles = StyleSheet.create({
  contentContainer: { flex: 1, alignItems: 'center', justifyContent: 'center' },
  settingsContainer: {},
  title: {
    fontSize: 25,
  },
});

export default SettingsScreen;
