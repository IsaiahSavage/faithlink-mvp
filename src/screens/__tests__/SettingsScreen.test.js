import React from 'react';
import { render } from '@testing-library/react-native';
import SettingsScreen from '../SettingsScreen'

describe('PrivacyPolicy', () =>{
    //see if it renders
    it('page renders correectly', () =>{
       render(<SettingsScreen />); 
    });

    //is not changing between loads
    it('matches the snapshot', () => {
        const { toJSON } = render(<SettingsScreen />);
        expect(toJSON()).toMatchSnapshot();
      });

    //text matches
    it('text is the same', () =>{
        const {getByText} = render(<SettingsScreen />);
        const title = getByText(/Settings coming soon!/);
        expect(title).toBeTruthy();
    });
});