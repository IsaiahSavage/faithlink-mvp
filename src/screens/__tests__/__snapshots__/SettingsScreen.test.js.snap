// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PrivacyPolicy matches the snapshot 1`] = `
<View
  style={
    {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text
    style={
      {
        "fontSize": 25,
      }
    }
  >
    Settings coming soon!
  </Text>
</View>
`;
